### PR TITLE
Rules: schedule dunning and schedule payment actions

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

**Comment by slavcodev** [comment]
Screenshot

**Comment by slavcodev** [comment]
Please review, it is ready to merge now.

**Comment by slavcodev** [comment]
**Schedule payment**

screen
screen
screen

File: spec/definitions/Rules/ScheduledPayment.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+type: object
+properties:
+  # TODO: Consider to change name, e.q. scheduleInstruction, schedulePolicy
+  schedule:
+    $ref: "#/definitions/ScheduledPaymentInstruction"
+  # TODO: Consider to change name, e.q. amountInstruction, amountPolicy
+  amount:
+    type: string
+    enum:
+      - AIT
```
Comment (adamaltman) [inline spec/definitions/Rules/ScheduledPayment.yaml]: These string constants don't make sense.  Let's use something that someone can read and understand.  I think one is `balance-outstanding`... and the other is `invoice-total`.  

As far as the property names go, I think we should consider a better Schedule object.  I like `scheduleInstruction` but have you considered `scheduleWhen`?

Comment (slavcodev) [inline spec/definitions/Rules/ScheduledPayment.yaml]: > I think we should consider a better Schedule object

wondering, what improvements do you have in mind.

**Comment by slavcodev** [comment]
I'm going to merge this. Will continue in parent PR #24

**Pull request merged**